run-checks

```diff
@@ -110,19 +110,15 @@ if [ ! -z "$STATIC" ]; then
         exit 1
     fi
 
-    (
-        # pot file
-        echo Checking translations
-        TMPF="$(mktemp)"
-        addtrap "rm -f $TMPF"
-        ./update-pot "$TMPF"
-        if ! diff -u --ignore-matching-lines=.*POT-Creation-Date.* po/snappy.pot $TMPF; then
-            echo "You need to run ./update-pot"
-            exit 1
-        fi
-        # This is a subshell so we don't overwrite the all-important outer trap.
-    )
-
+    # pot file
+    echo Checking translations
+    TMPF="$(mktemp)"
+    addtrap "rm -f $TMPF"
```

Comment:
Strictly speaking, if the value of TMPF needs quoting, it needs quoting in there also.